### PR TITLE
[Instana Exporter] Wrong JSON format for serialized spans

### PR DESCRIPTION
**Pull request by zivaninstana**
Fixes #.

## Changes

Instana backend reported generated JSON by InstanaExporter as wrong. 
JSON should contains spans node as a root but it was not a case.

For significant contributions please make sure you have completed the following items:

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Instana/Implementation/Transport.cs [hunk 1]
```diff
@@ -26,8 +26,9 @@ namespace OpenTelemetry.Exporter.Instana.Implementation;
 
 internal class Transport
 {
-    private const int MultiSpanBufferLimit = 4096000;
-
+    private const int MULTI_SPAN_BUFFER_SIZE = 4096000;
```
Comment (Kielek) [inline src/OpenTelemetry.Exporter.Instana/Implementation/Transport.cs:29]: Could you please explain why this difference between buffer size and used limit?

File: src/OpenTelemetry.Exporter.Instana/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* Wrong JSON for serialized spans was generated after last changes fixed
```
Comment (Kielek) [inline src/OpenTelemetry.Exporter.Instana/CHANGELOG.md:5]:
```suggestion
* Fixes issue in span serialization process introduced in 1.0.2 version.
  ([#979](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/979))
```

File: src/OpenTelemetry.Exporter.Instana/OpenTelemetry.Exporter.Instana.csproj [hunk 3]
```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
 	<PackageTags>Instana Tracing APM</PackageTags>
+	  <LangVersion>latest</LangVersion>
```
Comment (Kielek) [inline src/OpenTelemetry.Exporter.Instana/OpenTelemetry.Exporter.Instana.csproj:6]: Common lang version is defined under https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/9877a3220a3977dce2fb29d7dc2742dee553a0a8/build/Common.props#L3

I think, it will be fine to create separate PR with bumping it to 11.0.
Please revert change here.

File: src/OpenTelemetry.Exporter.Instana/Implementation/Transport.cs [hunk 4]
```diff
@@ -26,8 +26,9 @@ namespace OpenTelemetry.Exporter.Instana.Implementation;
...
+    private const int MULTI_SPAN_BUFFER_LIMIT = 4070000;
+    private readonly byte[] tracesBuffer = new byte[MULTI_SPAN_BUFFER_SIZE];
```
Comment (Kielek) [inline src/OpenTelemetry.Exporter.Instana/Implementation/Transport.cs:31]: Please fix compilation errors.
(It should be easily reproducible locally if you switch Debug to Release).

File: src/OpenTelemetry.Exporter.Instana/Implementation/Transport.cs (hunk 1 above)
Comment (zivaninstana) [reply src/OpenTelemetry.Exporter.Instana/Implementation/Transport.cs:29]: Buffer size is a size of byte array but we should never reach that size, we should stop before, because of that we have limit config.

File: src/OpenTelemetry.Exporter.Instana/OpenTelemetry.Exporter.Instana.csproj (hunk 3 above)
Comment (zivaninstana) [reply src/OpenTelemetry.Exporter.Instana/OpenTelemetry.Exporter.Instana.csproj]: Ok, I will

File: src/OpenTelemetry.Exporter.Instana/Implementation/Transport.cs (hunk 4 above)
Comment (zivaninstana) [reply src/OpenTelemetry.Exporter.Instana/Implementation/Transport.cs]: 👍

**Review by Kielek — APPROVED** [review]
